LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -71,6 +71,8 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
         if shouldBecomeFirstResponder && !context.coordinator.didBecomeFirstResponder {
             textField.becomeFirstResponder()
             context.coordinator.didBecomeFirstResponder = true
+        } else if !shouldBecomeFirstResponder && context.coordinator.didBecomeFirstResponder {
+            context.coordinator.didBecomeFirstResponder = false
```

Comment:
Explicitly not resigning first responder here, since the API's purpose is just to be able to make a text field active on command, not do real management of the responder chain.